rhobs/alerting/data_plane/prometheus.stability_image_repository_provision_failures_alerts.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-stability-image-repository-provision-failures-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: stability-image-repository-provision-failures
+    interval: 1m
+    rules:
+    - alert: StabilityImageRepositoryProvisionFailures
+      expr: |
+# todo
```

Comment:
The CI `check-and-test-prometheus-rules` task is failing because this file isn't proper yaml. I suspect this line is the issue.

```
time="2024-01-09T21:57:58Z" level=fatal msg="[rhobs/alerting/data_plane/prometheus.stability_image_repository_provision_failures_alerts.yaml] does not store a Kube object serialized in YAML: yaml: line 11: did not find expected key\n"
make: *** [Makefile:23: check-and-test] Error 1
```

Reply:
yes, `yamllint` says the yaml is broken.

```
stdin
  1:1       warning  missing document start "---"  (document-start)
  9:3       error    wrong indentation: expected 4 but found 2  (indentation)
  12:5      error    wrong indentation: expected 6 but found 4  (indentation)
  15:9      error    syntax error: expected <block end>, but found '<scalar>' (syntax)
  31:81     error    line too long (120 > 80 characters)  (line-length)
  33:81     error    line too long (143 > 80 characters)  (line-length)
```